packages/desktop/electron/main.js

```diff
@@ -175,14 +175,27 @@ if (app.isPackaged) {
     paths.errorHtml = path.join(__dirname, '../error.html')
 }
 
+/**
+ * Check URL against allowlist
```

Comment:
Comment is wrong. A propos, why are we following the block list instead of an allow list? Allow lists are much safer.